### PR TITLE
Create subnet tx

### PR DESCRIPTION
**Pull request by Ericson2314**

### CODE REVIEW

File: tests/p-chain-tests.ts [hunk 1]
```diff
@@ -381,6 +381,80 @@ describe('Staking tests', async function () {
...
+      0xbe, 0x82, 0xec, 0xc0, 0x0c, 0x34, 0xf3, 0x61,
+      0xed, 0xa8, 0xeb, 0x30, 0xfb, 0x5a, 0x71, 0x5c,
+      // number of inputs
+      0x00, 0x00, 0x00, 0x01,
+      // input
+      0xdf, 0xaf, 0xbd, 0xf5, 0xc8, 0x1f, 0x63, 0x5c,
+      0x92, 0x57, 0x82, 0x4f, 0xf2, 0x1c, 0x8e, 0x3e,
+      0x6f, 0x7b, 0x63, 0x2a, 0xc3, 0x06, 0xe1, 0x14,
+      0x46, 0xee, 0x54, 0x0d, 0x34, 0x71, 0x1a, 0x15,
+      // addresses?
```
Comment (patrick-ogrady) [inline tests/p-chain-tests.ts]: what is the `?`?

File: tests/p-chain-tests.ts [hunk 2]
```diff
@@ -381,6 +381,80 @@ describe('Staking tests', async function () {
...
+      0xed, 0xa8, 0xeb, 0x30, 0xfb, 0x5a, 0x71, 0x5c
+    ]);
+
+    const pathPrefix = "44'/9000'/0'";
+    const pathSuffixes = ["0/0", "0/1", "100/100"];
+    // Need to add headers for SubnetID and Sigindices?
+    const prompts = chunkPrompts([
+      {header: 'Sign', body: 'Create Subnet'},
+      {header: 'Threshold', body: '1'},
+      {header: 'Rewards To', body: 'local1mg47uqd7stkvqrp57ds7m28txra45u2uzkta8n' },
```
Comment (patrick-ogrady) [inline tests/p-chain-tests.ts]: There are no rewards.

Comment (kmenon01) [reply tests/p-chain-tests.ts]: Should this be Owner Address?

File: tests/p-chain-tests.ts (hunk 1 above)
Comment (kmenon01) [reply tests/p-chain-tests.ts]: Not sure why the question mark is there, will get rid of it

File: tests/p-chain-tests.ts [hunk 3]
```diff
@@ -381,6 +379,79 @@ describe('Staking tests', async function () {
...
+      0xbe, 0x82, 0xec, 0xc0, 0x0c, 0x34, 0xf3, 0x61,
+      0xed, 0xa8, 0xeb, 0x30, 0xfb, 0x5a, 0x71, 0x5c
+    ]);
+
+    const pathPrefix = "44'/9000'/0'";
+    const pathSuffixes = ["0/0", "0/1", "100/100"];
+    // Need to add headers for SubnetID and Sigindices?
+    const prompts = chunkPrompts([
+      {header: 'Sign', body: 'Create Subnet'},
+      {header: 'Threshold', body: '1'},
```
Comment (patrick-ogrady) [inline tests/p-chain-tests.ts:441]: I think we may also want to print the number of addresses here?

Comment (patrick-ogrady) [reply tests/p-chain-tests.ts:441]: Or, we can assume it is always 1 address and then just hide threshold.

Comment (patrick-ogrady) [reply tests/p-chain-tests.ts:441]: (I would guess that's what we want)

Comment (patrick-ogrady) [reply tests/p-chain-tests.ts:441]: Disregard

**Review by patrick-ogrady — APPROVED** [review]
LGTM

**Pull request merged**